Tests/MongoSwiftSyncTests/MongoDatabaseTests.swift

```diff
@@ -175,6 +175,75 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
         expect(listNamesEvents[2].command["nameOnly"]).to(equal(false))
         expect(listNamesEvents[3].command["nameOnly"]).to(equal(false))
     }
+
```

Comment:
@kmahar can the unified tests for this be unskipped without any other changes?

Reply:
I don't think there are any other changes to make here. The only things to change would be code that is introduced in the PR I just opened. Since I assume this PR is probably going to get merged before mine, I will rebase on top of this and update accordingly once this goes in. It should just require adding support for a database target to the `UnifiedAggregate` operation, and unskipping a test.